src/utils/collateral.ts

```diff
@@ -63,3 +66,72 @@ export const formatEscrowDeposit = (tokenInfo: TokenView): EscrowDeposit => {
   } as EscrowDeposit;
   return escrowDeposit;
 };
+
+export const formatCollateralRevenue = (
+  spigot: AggregatedSpigot,
+  reserves: {
+    [tokenAddress: string]: {
+      unusedTokens: string;
+      ownerTokens: string;
+      operatorTokens: string;
+    };
+  },
+  tokenPrices: {
+    [address: string]: BigNumber;
+  }
+): AggregatedSpigot => {
+  console.log(spigot);
+  const { revenueValue, revenueSummary, ...rest } = spigot;
+  const [newRevenueValue, newRevenueSummary]: [BigNumber, RevenueSummaryMap] = Object.values(
+    revenueSummary
+  ).reduce<any>(
+    (agg, { token, amount, ...summary }) => {
+      const checkSumAddress = ethers.utils.getAddress(token.address);
+      console.log(token.address);
+      const usdcPrice = tokenPrices[checkSumAddress] ?? BigNumber.from(0);
+      // const amountLessOperatorTokens = BigNumber.from(amount).sub(
+      //   BigNumber.from(reserves[checkSumAddress]?.operatorTokens ?? BigNumber.from(0))
+      // );
+      const amountLessOperatorTokens = BigNumber.from(reserves[checkSumAddress]?.ownerTokens ?? BigNumber.from(0));
```

Comment:
why not name var `ownerTokens`?

Reply:
changed the name from `amountLessOperatorTokens` to `ownerTokens`